### ISSUE TITLE
Linting requires eslint-plugin-jsx-a11y, not in package.json

### ISSUE THREAD

**Issue by kyleshevlin** [lelandrichardson/redux-pack#5]
Forked the repo. Ran:

```
npm install
```
then:
```
npm run test
```
Got an error that boils down to:
```
ESLint couldn't find the plugin "eslint-plugin-jsx-a11y".
```

My guess is that you have it installed globally on your machine, as I didn't see it in the package.json. Thought this might be something you wanted to add as a devDependency.

**Comment by kyleshevlin** [comment]
In an effort to provide more detail, I have installed local to the project eslint-plugin-jsx-a11y. After doing so, it errored asking for the eslint-plugin-react package. I installed that next. I then ran `npm run test` once more and received the following error:

```
Referenced from: airbnb
Referenced from: /Library/WebServer/Documents/Work/redux-pack/.eslintrc
Error: /Library/WebServer/Documents/Work/redux-pack/node_modules/eslint-config-airbnb/rules/react-a11y.js:
        Configuration for rule "jsx-a11y/anchor-has-content" is invalid:
        Value "" is the wrong type.
```

In addition, a fresh `npm install` with this package.json returns a:
```
npm ERR! peer dep missing: eslint-plugin-jsx-a11y@^2.2.2, required by eslint-config-airbnb@12.0.0
```

**Comment by lelandrichardson** [comment]
Fixed! Sorry about that.
